pygments/lexers/scripting.py

```diff
@@ -1253,13 +1253,13 @@ class MiniScriptLexer(RegexLexer):
                 'in', 'isa', 'then', 'repeat', 'return', 'while'), suffix=r'\b'),
              Keyword),
             (words((
-            	'abs', 'acos', 'asin', 'atan', 'ceil', 'char', 'cos', 'floor',
-            	'log', 'round', 'rnd', 'pi', 'sign', 'sin', 'sqrt', 'str', 'tan',
-            	'hasIndex', 'indexOf', 'len', 'val', 'code', 'remove', 'lower',
-            	'upper', 'replace', 'split', 'indexes', 'values', 'join', 'sum',
-            	'sort', 'shuffle', 'push', 'pop', 'pull', 'range',
-            	'print', 'input', 'time', 'wait', 'locals', 'globals', 'outer',
-            	'yield'), suffix=r'\b'),
+                'abs', 'acos', 'asin', 'atan', 'ceil', 'char', 'cos', 'floor',
+                'log', 'round', 'rnd', 'pi', 'sign', 'sin', 'sqrt', 'str', 'tan',
+                'hasIndex', 'indexOf', 'len', 'val', 'code', 'remove', 'lower',
+                'upper', 'replace', 'split', 'indexes', 'values', 'join', 'sum',
+                'sort', 'shuffle', 'push', 'pop', 'pull', 'range',
+                'print', 'input', 'time', 'wait', 'locals', 'globals', 'outer',
+                'yield'), suffix=r'\b'),
```

Comment:
I assume this is a drive-by shooting that the formatting changed here?

Reply:
yes